### PR TITLE
Compute and store SUPER-PD MDS-UPDRS scores

### LINKED ISSUE #126: score super-pd updrs scores

### PR DESCRIPTION
**Pull request by philerooski**
Resolves #126

### CODE REVIEW

File: compute_updrs_score/compute_updrs_score.R [hunk 1]
```diff
@@ -56,18 +57,75 @@ get_ahpd_clinical_data <- function(){
...
+    super_physician_off <- get_super_physician_scores(field_mapping, "Physician_OFF")
+    store_to_synapse(super_physician_off, "mdsupdrs_physician_off_med_scores.csv", "syn18691015")
     ahpd_updrs_scores <- get_ahpd_clinical_data() %>%
         map_column_names("ahpd") %>%
         run_updrs_scoring(
             join_cols = c("guid", "createdOn", "C_ONLDOPA")) %>%
-        write.table(OUTPUT_FILENAME , sep = "\t", row.names=F, quote=F)
+        write.table(OUTPUT_FILENAME, sep = "\t", row.names=F, quote=F)
     #' store result to synapse
     f <- File(OUTPUT_FILENAME, SYNAPSE_PARENT_ID)
```
Comment (arytontediarjo) [inline compute_updrs_score/compute_updrs_score.R:125]: A bit trivial: I would refactor store_to_synapse(..) to handle file uploads

**Review by arytontediarjo — APPROVED** [review]
Looks good! I will refactor store_to_synapse(..) on another PR so we are not doing duplicate operations. Is the strictly have to be in .csv format for the results?

**Comment by philerooski** [comment]
No need to change anything. It works as-is and its intended use is so specific it's not worth generalizing the code any further. I just wanted you to take a look!

**Comment by arytontediarjo** [comment]
coooool. all good then

**Pull request merged**